moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -168,7 +175,16 @@ const ComponentForm = ({
             />
           </Grid>
         )}
-
+        <Grid item xs={12}>
+          <ControlledAutocomplete
+            id="work_type"
+            label="Work Type(s)"
+            multiple
+            options={workTypeOptions}
+            name="work_types"
+            control={control}
+          />
+        </Grid>
```

Comment:
I believe we want to require always one work type? and need to make sure that the `New` work type is available for every component? and would be nice to default `New` since clicking it is annoying?